### PR TITLE
patron: adapt profile view for new 'circulation module'

### PR DESCRIPTION
**Pull request by AoNoOokami**
* Changes loan.item_pid into loan.item_pid.value.

Co-Authored-by: Alicia Zangger <alicia.zangger@rero.ch>

## Why are you opening this PR?

To adapt patron profile view to the new "circulation module".
Task 1657 of US1489

https://tree.taiga.io/project/rero21-reroils/task/1657?kanban-status=1224894
https://tree.taiga.io/project/rero21-reroils/us/1489?milestone=269576

## Dependencies

My PR depends on the following `rero-ils-ui`'s PR(s):

No dependency

## How to test?

`bootstrap` & ` setup`
Log in as a patron and go to patron profile view.

## Code review check list

- [ ] Commit message template compliance.
- [ ] Commit message without typos.
- [ ] File names.
- [ ] Functions names.
- [ ] Functions docstrings.
- [ ] Unnecessary commited files?

### CODE REVIEW

File: rero_ils/modules/loans/api.py [hunk 1]
```diff
@@ -560,7 +560,7 @@ def _process_patron_profile_fees(patron, organisation, status='open'):
         if 'loan' in transaction:
             item_pid = Loan.get_record_by_pid(transaction.loan.pid)\
                 .get('item_pid')
-            item = Item.get_record_by_pid(item_pid)
+            item = Item.get_record_by_pid(item_pid.get('value'))
```
Comment (rerowep) [inline rero_ils/modules/loans/api.py:563]: There is something wrong elsewhere if you have to use `item_pid.get('value')`.
`item_pid` should be already a pid value and not a dictionary.
In the loan json schema this is declared as string value!

Comment (AoNoOokami) [reply rero_ils/modules/loans/api.py:563]: This comes form invenio-circulation

Comment (AoNoOokami) [reply rero_ils/modules/loans/api.py:563]: https://github.com/inveniosoftware/invenio-circulation/blob/c3fcc1a4674d5df24221e3c068e8e0c07db68016/invenio_circulation/mappings/v5/loans/loan-v1.0.0.json#L39

Comment (blankoworld) [reply rero_ils/modules/loans/api.py:563]: `item_pid` is a loan in this context. And the loan jsonschema have properties type and value here: https://github.com/rero/rero-ils/blob/US1394-invenio-circulation/rero_ils/modules/loans/mappings/v6/loans/loan-ils-v0.0.1.json#L51

**Pull request merged**